resources/views/page.blade.php

```diff
@@ -10,7 +10,7 @@
     'boxed' => 'layout-boxed',
     'fixed' => 'fixed',
     'top-nav' => 'layout-top-nav'
-][config('adminlte.layout')] : ''))
+][config('adminlte.layout')] : '') . (config('adminlte.collapse_sidebar') ? 'sidebar-collapse' : ''))
```

Comment:
This will not work correctly: e.g. when the layout is `fixed`, then we get `fixedsidebar-collapse`.